Dockerfile

```diff
@@ -91,7 +75,7 @@ ENV FSLOUTPUTTYPE=NIFTI_GZ
 
 #############################################
 # Download and install Connectome Workbench
-RUN apt-get update && apt-get -y install connectome-workbench=1.2.3-1~nd14.04+1
+RUN apt-get update && apt-get -y install connectome-workbench
```

Comment:
Which version of the connectome-workbench is installed without this specification? I'm wondering how we can be certain that this will not have unintended/unexpected affects on the results.